plugins/generator-1.19.2/forge-1.19.2/templates/musicdisc.java.ftl

```diff
@@ -29,6 +29,7 @@
 -->
 
 <#-- @formatter:off -->
+<#include "procedures.java.ftl">

```

Comment:
Are you sure this include is needed? The new macro is in triggers. Please check other files too

Reply:
Done!